File: .tmp_sc4013/java_22/3963_create_grid_with_exactly_one_path.java
// CONFIG class=Solution method=createGrid types=None
// LeetCode 3963 - Create Grid With Exactly One Path
// https://leetcode.com/problems/create-grid-with-exactly-one-path/

class Solution {
    public String[] createGrid(int m, int n) {
        String[] g = new String[m];
        for (int i = 0; i < m; i++) {
            char[] row = new char[n];
            for (int j = 0; j < n; j++) row[j] = '#';
            if (i == 0) for (int j = 0; j < n; j++) row[j] = '.';
            row[n - 1] = '.';
            g[i] = new String(row);
        }
        return g;
    }
}

File: .tmp_rs4013/rust_check22/3963_create_grid_with_exactly_one_path.rs
struct Solution;
// LeetCode 3963 - Create Grid With Exactly One Path
// https://leetcode.com/problems/create-grid-with-exactly-one-path/

impl Solution {
    pub fn create_grid(m: i32, n: i32) -> Vec<String> {
        let mut g = vec![vec![b'#'; n as usize]; m as usize];
        for j in 0..n as usize {
            g[0][j] = b'.';
        }
        for i in 0..m as usize {
            g[i][n as usize - 1] = b'.';
        }
        g.into_iter()
            .map(|row| String::from_utf8(row).unwrap())
            .collect()
    }
}

fn main() {}
